lms/startup.py

```diff
@@ -45,7 +45,7 @@ def run():
     # Initialize Segment.io analytics module. Flushes first time a message is received and
     # every 50 messages thereafter, or if 10 seconds have passed since last flush
     if settings.FEATURES.get('SEGMENT_IO_LMS') and hasattr(settings, 'SEGMENT_IO_LMS_KEY'):
-        analytics.init(settings.SEGMENT_IO_LMS_KEY, flush_at=50)
+        analytics.write_key = settings.SEGMENT_IO_LMS_KEY
```

Comment:
Correct.  1.1.0 only expects `write_key` to be set.  And flushing is now asynchronous so there is no need (or way to set `flush_at` anymore.  If we need to ensure that events are flushed (say before we terminate a process) we can call it directly with `analytics.flush()`